test/pageobjects/Login.page.js

```diff
@@ -0,0 +1,50 @@
+
+import BasePage from '../pageobjects/base.page'
+
+class Login extends BasePage{
+
+    //WebElements
+    get principal(){return $('.align-content')}
+    get tittle(){return $('.login-logo')}
+    get usernamein(){ return $('#username') }
+    get passwordin(){return $('#password')}
+    get rememberme(){return $('#remember-me')}
+    get signin(){return $('#submit')}
+    get signuphere(){return $('=Sign Up Here')}
+    get flash() {return $('.badge')}
+
+    /**Metodos para iniciar sesion en Digital bank
+     * @param {String} user
+     * @param {String} password
+     * 
+     */
+    async loguearclick(username,password){
+        await this.vaciarCampoYEnviarTexto(await this.usernamein,username)
+        await this.vaciarCampoYEnviarTexto(await this.passwordin,password);
+        await this.clickearElemento(await this.signin);
+        
+    }
+
+   /*
+    async loguearenter(username,password){
+        await this.usernamein(username).keys('tab')
+        await this.passwordin(password).keys('tab')
+        await this.signin.keys('enter')
+    }*/
+
+    async loguearclickreme(username,password){
+        await this.vaciarCampoYEnviarTexto(await this.usernamein,username)
```

Comment:
Este metodo repite varios pasos del de loguearse con click en el boton. Quizas se puede descomponer las operaciones un poco mas para evitar la duplicacion: Se me ocurre:

- Crear un metodo para la parte que "completar credenciales", que ambos metodos comparten
- Un metodo para hacer click en el boton de sign in
- Un metodo para hacer click en el checkbox de rememberme

Luego en la definicion del test se usan los metodos necesarios para cada comportameinto (ej, en el caso de loguearse con click, se usaran "completar credenciales", y "click en boton de sign in", mientras que para el de loguearse con "remember me", entre "completar credenciales" y "click en boton sign in" se llamaria a "click en checkbox de remember me")